mlir/lib/Dialect/OpenACC/IR/OpenACC.cpp

```diff
@@ -815,6 +815,11 @@ LogicalResult acc::ParallelOp::verify() {
   if (failed(checkWaitAndAsyncConflict<acc::ParallelOp>(*this)))
     return failure();
 
+  if (getCombined().has_value() &&
+      getCombined().value() != acc::CombinedConstructsType::ParallelLoop) {
```

Comment:
nit: it the rest of the file we tend to use this form
```suggestion
  if (getCombined() && *getCombined() != acc::CombinedConstructsType::ParallelLoop) {
```

Reply:
Thanks! Will do :)

Reply:
Replaced with UnitAttr - so got rid of this logic completely.